src/plugins/vis_builder/public/application/components/option.tsx

```diff
@@ -20,6 +20,7 @@ export const Option: FC<Props> = ({ title, children, initialIsOpen = false }) =>
         buttonContent={title}
         className="vbOption"
         initialIsOpen={initialIsOpen}
+        data-test-subj={`vbOption-${title.replace(/\s+/g, '-')}`}
```

Comment:
super minor nit, but don't we already have a standard test utility for formatting test subjects like this?

Reply:
Do you know where that is? I couldn't find it in my scan of the repo.

Reply:
FYI: this is how its done on dashboards: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/dashboard/public/application/listing/dashboard_listing.js#L176

Reply:
Yeah, I guess not 😞